core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -81,9 +81,6 @@ describe('Topics and skills dashboard functionality', function() {
     topicsAndSkillsDashboardPage.expectNumberOfTopicsToBe(0);
   });
 
-  // TODO(aks681): Once skill editor tests are done by Nalin, add tests here
```

Comment:
Per the TODO ... weren't some tests supposed to be added here?

Reply:
Yeah, the tests were added, but in skillEditor.py, as skill publishing seemed like a skill editor task. The check in topics and skills dashboard about whether it was moved to new section is also done there.